janusgraph-dist/src/pkg/static/pkgcommon/payload/etc/janusgraph/janusgraph-env.sh

```diff
@@ -207,10 +207,7 @@ JVM_OPTS="$JVM_OPTS -XX:MaxTenuringThreshold=1"
 JVM_OPTS="$JVM_OPTS -XX:CMSInitiatingOccupancyFraction=75"
 JVM_OPTS="$JVM_OPTS -XX:+UseCMSInitiatingOccupancyOnly"
 JVM_OPTS="$JVM_OPTS -XX:+UseTLAB"
-# note: bash evals '1.7.x' as > '1.7' so this is really a >= 1.7 jvm check
-if [ "$JVM_VERSION" \> "1.7" ] && [ "$JVM_ARCH" = "64-Bit" ] ; then
-    JVM_OPTS="$JVM_OPTS -XX:+UseCondCardMark"
-fi
+JVM_OPTS="$JVM_OPTS -XX:+UseCondCardMark"
```

Comment:
Is the x64 architecture check no longer necessary with Java 8?

Reply:
Good catch.  Appears it's only valid for `-server` option which is only valid on 64-bit.